node/src/arrow.ts

```diff
@@ -142,15 +144,18 @@ export function makeArrowTable (
   // TODO: sample dataset to find missing columns
   const columnNames = Object.keys(data[0])
   for (const colName of columnNames) {
-    const values = data.map((datum) => datum[colName])
+    let values = data.map((datum) => datum[colName])
     let vector: Vector
 
     if (opt.schema !== undefined) {
       // Explicit schema is provided, highest priority
-      vector = vectorFromArray(
-        values,
-        opt.schema?.fields.filter((f) => f.name === colName)[0]?.type
-      )
+      const fieldType: DataType | undefined = opt.schema.fields.filter((f) => f.name === colName)[0]?.type as DataType
+      if (fieldType instanceof Int64) {
+        // wrap in BigInt to avoid bug: https://github.com/apache/arrow/issues/40051
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
+        values = values.map((v) => BigInt(v)) as any[]
```

Comment:
This is a minor nit, don't stress much over it.

Why is the `as any` needed here?  I would think `values` is already `any[]` and so you should be able to assign to it.  If it isn't, then maybe it would be more clear to change the above line?

```
let values = data.map((datum) => datum[colName]) as any[]
```

Reply:
I can try this. I really didn't understand the errors eslint was giving me...

Reply:
Okay turns out it wasn't necessary in either spot after all. 🤷